edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Service/hubStart.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+
+###############################################################################
+# Set up EdgeHub to run as a non-root user at runtime, if allowed.
+# 
+# If this script is started as root:
+#  1. It reads the EDGEHUBUSER_ID environment variable, default UID=13623.
+#  2. If the User ID does not exist as a user, create it.
+#  3. If "StorageFolder" env variable exists, use as basepath, else use /tmp
+#     Do same for backuppath
+#  4. If basepath/edgehub exists, make sure all files are owned by EDGEHUBUSER_ID
```

Comment:
I don't believe EdgeHub uses backup..

Reply:
There was this file: https://github.com/Azure/iotedge/blob/master/edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Service/Constants.cs#L9
So I supposed it has one.

Reply:
Hmm, yes you are right actually - I forgot about this one. It is used for the backup and restore feature...